lib/pager_duty/connection.rb

```diff
@@ -86,6 +85,12 @@ class ParseTimeStrings < Faraday::Response::Middleware
         service
       )
 
+      COLLECTION_KEYS = %w(
```

Comment:
Would this be a little more informative if we called this `NESTED_COLLECTION_KEYS`?